src/components/OSCALControl.js

```diff
@@ -94,7 +94,6 @@ export default function OSCALControl(props) {
               isEditable={props.isEditable}
               onFieldSave={props.onFieldSave}
               partialRestData={props.partialRestData}
-              restPath={props.restPath}
```

Comment:
OSCAL object renders shouldn't need to be passed a `restPath`.  The component knows what `oscalObjectType` it's rendering and can pass that to REST functions if needed.